x-pack/plugins/observability/server/services/slo/delete_slo.test.ts

```diff
@@ -33,6 +33,7 @@ describe('DeleteSLO', () => {
 
       await deleteSLO.execute(slo.id);
 
+      expect(mockRepository.findById).toHaveBeenCalledWith(slo.id);
```

Comment:
Don't we expect the SLO id here? as we are calling `deleteSLO.execute(slo.id);`.

Reply:
I'm not sure to follow the question. We are expecting findById to be called with the slo.id
Are you suggesting we shouldn't?

Reply:
It should have the SLO id and my comment was about keeping the SLO id. My bad I maybe missed read it